metricflow/model/model_validator.py

```diff
@@ -32,7 +32,7 @@
 class ModelValidator:
     """A Validator that acts on UserConfiguredModel"""
 
-    VALIDATION_RULES: List[ModelValidationRule] = [
+    DEFAULT_RULES: List[ModelValidationRule] = [
```

Comment:
Let's make this a Tuple so it can't be accidentally mutated in the runtime...

Reply:
Oooo, yes! Good call!